documentation/modules/rn-27-resolved-issues.adoc

```diff
@@ -7,6 +7,22 @@
 
 {project-first} 2.7 has the following resolved issues:
 
+[id="resolved-issues-2-7-9_{context}"]
+== Resolved issues 2.7.9
+
+.Forklift controller panic in the inventory
+
+In earlier releases of {project-short}, the `forklift-contoller` inventory container would panic, and the VM would not be migrated. This issue has been resolved in {project-short} 2.7.9. link:https://issues.redhat.com/browse/MTV-1610[(MTV-1610)]
+
+.OVN secondary network not working with Multus default network override
+
+In earlier releases of {project-short}, the OVN secondary network was not working with Multus default network override. This issue caused the importer pod to become stuck, as they were created when there were multiple networks and the wrong default network override annotation was configured. This issue has been resolved in {project-short} 2.7.9. link:https://issues.redhat.com/browse/MTV-1645[(MTV-1645)]
```

Comment:
Does "they" refer to the "importer pod"? Might want to use the phrase rather than the pronoun.